### ISSUE TITLE
Messages not showing on Android

### ISSUE THREAD

**Issue by RoshanNepal** [leegeunhyeok/react-native-zendesk-messaging#1]
Works fine on iOS 

![IMG_20230123_145336](https://user-images.githubusercontent.com/43002639/214004196-06b6d3f1-7cdb-4cf0-978a-90e6207264fb.jpg)
![IMG_20230123_150107](https://user-images.githubusercontent.com/43002639/214004208-c90dabea-da9c-4ec1-b81c-398f280aba48.png)

**Comment by leegeunhyeok** [comment]
Did you tried call `openMessagingView()` after some time? In my case, works fine. (maybe dependent on network status/server)

Sometimes messages not showing, but any errors not occurs from SDKs.
As you know, this library implemented React Native bridges only. Key features are SDKs function. (message load, showing and etc..)

Wipe your application data or if you logged in to Zendesk with `login()`, try another user token.

To summarize, the following is the case:

- calling `openMessagingView()` after some time.
- Wipe application data
- Try using another user token

**Comment by RoshanNepal** [comment]
Here's my implementation

`import React from 'react';
import {Text, View, TouchableOpacity} from 'react-native';
import * as Zendesk from 'react-native-zendesk-messaging';

const App = () => {
  React.useEffect(() => {
    Zendesk.initialize({channelKey: '***'});
  }, []);

  const _onPress = () => {
    try {
      Zendesk.openMessagingView();
    } catch (_) {
      //
    }
  };

  return (
    <View>
      <TouchableOpacity onPress={_onPress}>
        <Text>Support</Text>
      </TouchableOpacity>
    </View>
  );
};

export default App;
`

**Comment by RoshanNepal** [comment]
Here's my implementation

```
import React from 'react';
import {Text, View, TouchableOpacity} from 'react-native';
import * as Zendesk from 'react-native-zendesk-messaging';

const App = () => {
  React.useEffect(() => {
    Zendesk.initialize({channelKey: '***'});
  }, []);

  const _onPress = () => {
    try {
      Zendesk.openMessagingView();
    } catch (_) {
      //
    }
  };

  return (
    <View>
      <TouchableOpacity onPress={_onPress}>
        <Text>Support</Text>
      </TouchableOpacity>
    </View>
  );
};

export default App;
```

**Comment by leegeunhyeok** [comment]
Still have same issue without any errors?

I tried to reproduce this situations, but works fine in my environments. (tested on real devices, emulators)
Need more information for resolve this issue.

I would appreciate it if you could check native SDKs logs.

(Check below guides for enable SDKs log)
- Android: https://developer.zendesk.com/documentation/zendesk-web-widget-sdks/sdks/android/getting_started/#logging
- iOS: https://developer.zendesk.com/documentation/zendesk-web-widget-sdks/sdks/ios/getting_started/#troubleshooting

If still same issue occurs, report error log and reopen issue please.

**Comment by vinitsood** [comment]
Hi @RoshanNepal,

Be aware that Android and iOS channelKeys. The look familiar but are different. Confirm that your android key is correct. I hade the same issue :)
